model/metrics/llm-metrics.yaml

```diff
@@ -0,0 +1,109 @@
+groups:
+  - id: metric.openai.chat_completions.tokens
+    type: metric
+    metric_name: llm.openai.chat_completions.tokens
+    brief: "Number of tokens used in prompt and completions."
+    instrument: counter
+    unit: "token"
+    stability: experimental
+    attributes:
+      - ref: llm.response.model
+        requirement_level: required
+      - ref: error.type
+        requirement_level:
+          conditionally_required: "if the operation ended in error"
+      - ref: llm.usage.token_type
+      - ref: server.address
+        requirement_level: required
+  - id: metric.openai.chat_completions.choices
+    type: metric
+    metric_name: llm.openai.chat_completions.choices
+    brief: "Number of choices returned by chat completions call"
+    instrument: counter
+    unit: "choice"
+    stability: experimental
+    attributes:
+      - ref: llm.response.model
+        requirement_level: required
+      - ref: error.type
+        requirement_level:
+          conditionally_required: "if the operation ended in error"
+      - ref: llm.response.finish_reason
+      - ref: server.address
+        requirement_level: required
+  - id: metric.openai.chat_completions.duration
```

Comment:
is there a chance we can unify some metrics for different llm systems?

Since we have a span for LLM request, we can have a similar metric which is `llm.request.duration` (or `llm.operation.duration`) and then individual LLM systems would be able to specify/override it without renaming it.

Eventually this might be proven to be way to complex and not quite usable, but it'd be a good and simple start. 

also, openai (or all llms) can have a special attribute such as `llm.operation` (check out [`messaging.operation`](https://github.com/open-telemetry/semantic-conventions/blob/main/docs/messaging/messaging-spans.md#operation-names)) which can be used a  dimension on the metric and serve as a low-cardinality span name.